src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/UpdateSubmission.java

```diff
@@ -15,6 +16,9 @@ public class UpdateSubmission {
     @JsonProperty("userSubmission")
     private OverseasEntitySubmissionDto userSubmission;
 
+    @JsonProperty("entityNumber")
+    private String entityNumber;
+
```

Comment:
Do we need to set the entityNumber here as its already a part of OverseasEntitySubmissionDto, same for dueDiligence and presenter

Reply:
due diligence and presenter are covered in another story but OE number we should do here as I don't think it was called out in  https://companieshouse.atlassian.net/browse/UAR-495